README.md

```diff
@@ -91,6 +91,21 @@ Currently, we support Windows (7 or later), Linux, and macOS. An installed versi
 
    For more information, run `retdec-decompiler.py` with `--help`.
 
+### FreeBSD
+
+1. There are currently no pre-built "ports" package for FreeBSD. You will have to build and install the decompiler by yourself. The process is described below.
+
+2. Prior to you building the decompiler, you may need to install the following packages and execute the following command:
+* `sudo pkg install cmake python37 bison git autotools`
+* `sudo ln -s /usr/local/bin/python3.7 /usr/local/bin/python3`
```

Comment:
As we need the other packages to run the decompiler i.e. python37 (python3) I thought it was more efficient to install all the packages required rather than using a separate command i.e. * `sudo pkg installl cmake` in build instructions. I can have a separate command in build and remove cmake and git which are needed to build it.

Reply:
For the other systems, the _Installation and use_ section contains only things you need to install to actually run the decompiler. Build dependencies are in _Build and Installation_.

Reply:
So I suggest splitting the requirements.